src/components/Table/GridTable.tsx

```diff
@@ -607,18 +563,6 @@ function renderVirtual<R extends Kinded>(
       }}
       // Pin/sticky both the header row(s) + firstRowMessage to the top
       topItemCount={(stickyHeader ? headerRows.length : 0) + (firstRowMessage ? 1 : 0)}
-      itemSize={(el) => {
```

Comment:
This goes away b/c we don't use `display: contents` anymore right?

Reply:
Yup, 100%.